.circleci/config.yml

```diff
@@ -193,6 +193,34 @@ jobs:
           channel: << pipeline.parameters.alerts-slack-channel >>
           template: basic_fail_1
 
+  e2e_environment_test:
+    docker:
+      - image: mcr.microsoft.com/playwright:v1.40.0-focal
+    circleci_ip_ranges: true # opt-in to jobs running on a restricted set of IPs
+    steps:
+      - run:
+          name: Clone E2E tests
+          command: |
+            git clone https://github.com/ministryofjustice/hmpps-community-accommodation-tier-2-ui.git .
+      - run:
+          name: Update npm
+          command: 'npm install -g npm@latest'
+      - node/install-packages
+      - run:
+          name: E2E Check
+          command: |
+            npm run test:e2e
```

Comment:
I'm hoping I can just switch it to this instead of `npm run test` 🤞

Reply:
Would we need to [provide env variables](https://github.com/ministryofjustice/hmpps-temporary-accommodation-ui/blob/main/.circleci/config.yml#L67) like CAS3?

Reply:
I'm hoping that [the same environment variables we set on the CircleCI project for the old job](https://app.circleci.com/settings/project/github/ministryofjustice/hmpps-community-accommodation-tier-2-ui/environment-variables) will be there in the same way 🤞